### PR TITLE
final blogpost for team Codeaholics

### PR DESCRIPTION
**Pull request by xh3n1**

### CODE REVIEW

**Comment by xh3n1** [comment]
feel free to review it, @alicetragedy :)

File: blog/_posts/2017-09-21-Our-IntoxiCODED-cloudy-summer.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+title: Our IntoxiCODED cloudy summer
```
Comment (v-thakkar) [inline blog/_posts/2017-09-21-Our-IntoxiCODED-cloudy-summer.md]: Hi, thanks for the PR. You have changed Gemfile.lock. Can you please revert that change? For the blogpost PR, it is preferrable to submit blog post related files only. There is a separate issue related to versions of dependency packages. You can have a look at it and can send a separate PR for the same, in case you want to :)

File: blog/_posts/2017-09-21-Our-IntoxiCODED-cloudy-summer.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+layout: post
+created_at: Thu September 21 2017
```
Comment (v-thakkar) [inline blog/_posts/2017-09-21-Our-IntoxiCODED-cloudy-summer.md]: Your blog post is scheduled on Tuesday September 26 2017. Can you please change a date everywhere?

File: blog/_posts/2017-09-21-Our-IntoxiCODED-cloudy-summer.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+Finally, we had the chance to meet our coaches Morris and Joas in person, our mentor joined us as well. We had four amazing days full of hacking and working on specific tasks on Nextcloud. It's very helpful when you have your coaches nearby so you can directly ask them and clarify in case of any doubts. In addition to meeting our coaches we also met many other contributors during our stay in Berlin. Fun fact: We knew most of them already by their GitHub usernames, that's why the first question we asked them was their github username. ;)
+
+We attended some talks related to the collaboration between Red Hat and Nextcloud, Collabora Online, how the Technical University of Berlin uses Nextcloud, Scalability Recommendations, Security etc.
+
+On the weekend, there were some lightning talks and workshops where we also shared [our experience](https://www.youtube.com/watch?v=OpMtsE_MICs) as RGSoC students with them (a lot of applauses in the end, yaaay)! It was a wonderful conference and a great opportunity to network with the community behind Nextcloud, which makes you feel more motivated to keep contributing.
+![image](/img/blog/2017/codeaholics_nextcloudconference.jpg)
```
Comment (v-thakkar) [inline blog/_posts/2017-09-21-Our-IntoxiCODED-cloudy-summer.md]: This image breaks because of the wrong pathname. Please fix it.

File: blog/_posts/2017-09-21-Our-IntoxiCODED-cloudy-summer.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+<div class="image-credits">Nextcloud Conference. (Photo:Raghu Nayyar)</div>
+And just for the record, coding while drinking Club Mate is one of the best feelings ever. (Try it :P)
```
Comment (v-thakkar) [inline blog/_posts/2017-09-21-Our-IntoxiCODED-cloudy-summer.md]: It would be better to have one line space between image and normat text [not image-credits though]. Can you please do that over here?

File: blog/_posts/2017-09-21-Our-IntoxiCODED-cloudy-summer.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+![gif](/img/blog/2017/codeaholics_clubmate.gif)
```
Comment (v-thakkar) [inline blog/_posts/2017-09-21-Our-IntoxiCODED-cloudy-summer.md]: This is not working. I dont think there is a separate gif tag in markdown. You would need to use image tag.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by v-thakkar** [comment]
@xh3n1 Can you send the requested changes as your blog post is scheduled to be published tomorrow? :)

**Comment by xh3n1** [comment]
hey @nerdyvaishali Thank your for your review. I made  the changes.

**Review by v-thakkar — APPROVED** [review]

File: blog/_posts/2017-09-26-Our-IntoxiCODED-cloudy-summer.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+We attended some talks related to the collaboration between Red Hat and Nextcloud, Collabora Online, how the Technical University of Berlin uses Nextcloud, Scalability Recommendations, Security etc.
+
+On the weekend, there were some lightning talks and workshops where we also shared [our experience](https://www.youtube.com/watch?v=OpMtsE_MICs) as RGSoC students with them (a lot of applauses in the end, yaaay)! It was a wonderful conference and a great opportunity to network with the community behind Nextcloud, which makes you feel more motivated to keep contributing.
+And just for the record, coding while drinking Club Mate is one of the best feelings ever. (Try it :P)
+
+
+![image](/img/blog/2017/codeaholics_nextcloudconference.jpg)
```
Comment (v-thakkar) [inline blog/_posts/2017-09-26-Our-IntoxiCODED-cloudy-summer.md]: Hi @xh3n1. You didn't fix the image path here. Please fix it.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Review by v-thakkar — APPROVED** [review]
